### ISSUE TITLE
leapp-cli: cannot add aws sso integration

### ISSUE THREAD

**Issue by nitrocode** [Noovolari/leapp#301]
**Describe the bug**
cannot add aws sso integration from leapp-cli

**Leapp Version**
0.13.0

**To Reproduce**

```
$ leapp integration create --integrationAlias=snip --integrationMethod=In-app --integrationPortalUrl=https://snip.awsapps.com/start/ --integrationRegion=us-east-1
$ leapp integration list
 Integration Name Portal URL                         Region    Status  Expiration
 ──────────────── ────────────────────────────────── ───────── ─────── ──────────
 snip             https://snip.awsapps.com/start/ us-east-1 Offline -
```

I then see the integration in Leapp, right click, click on Login, and the browser does not pop up.

**Expected behavior**
Allowed login from the Leapp app

**Screenshots**
N/A

**Additional context**
Adding an integration via clickops with the same settings in Leapp brings up the login in-app popup as expected.

**Comment by andreacavagna01** [comment]
added as a bug, thanks for reporting it! a blog series about how to script with CLI is coming. 

Could it be a great idea to open a discussion on how the scripting of the CLI should work?

**Comment by pethron** [comment]
Quick update: will be fixed in the upcoming release (0.13.3)
